index.html

```diff
@@ -271,4 +271,38 @@ <h3 class="font-style">Skills</h3>
 				</div>
 			</section>
 		</div>
-	</section>
\ No newline at end of file
+	</section>
+
+	<!-- add contact form -->
+  
+	<footer class="contactme" id="contactme">
+		<h4 class="head-contact">Contact me</h4>
+		<p>
+			If you have an application you are interested in developing, a feature
+			that you need built or a project that needs coding. I’d love to help
+			with it!
+		</p>
+    <form class="form-contact" action="https://formspree.io/f/xwkjdrpo" method="post">
+				<ul class="form-ul">
+					<li>

```

Comment:
- [ ] Please avoid using the `ul` inside the form. You can still use the form and with the help of `flexbox` you can easily attain the structure. As I noticed that you used the `flexbox` on your `ul` but you can optimize it more and make your code semantically more correct which makes your website more accessible. Whereas please note that `ul` and `li` are just meant for listing purposes so use it wisely 😄 . If you do the changes your code looks more leaner and easy to understand.